src/test/custom_loader_test.cpp

```diff
@@ -376,7 +376,7 @@ class TestCustomLoader : public ::testing::Test {
 class MockModelInstance : public ovms::ModelInstance {
 public:
     MockModelInstance(ov::Core& ieCore) :
-        ModelInstance("UNUSED_NAME", 42, ieCore) {}
+        ModelInstance("UNUSED_NAME", 42, ieCore, nullptr, nullptr) {}

```

Comment:
Maybe we could specify default registry here to make sure tests are incrementing metrics?
If registry, then maybe config as well.

Reply:
Not adding if not required.